morphir/src/org/finos/morphir/runtime/sdk/DecimalSDK.scala

```diff
@@ -24,10 +33,16 @@ object DecimalSDK {
       RTDecimal(result)
   }
 
+  val asString = DynamicNativeFunction1("toString") {
```

Comment:
the val name should be consistent

Reply:
toString is already defined for all objects in scala. I can do _toString or the likes if we want the name to look the same, but I can't override it, since the return value is different than the existing defined toString

Reply:
I forgot about that 😆 let's do `_toString`